Applications/Spreadsheet/SpreadsheetView.cpp

```diff
@@ -94,6 +100,20 @@ SpreadsheetView::SpreadsheetView(Sheet& sheet)
     };
 }
 
+void SpreadsheetView::switched_out() {
```

Comment:
c l a n g f o r m a t

Reply:
Huh.
That shouldn't have happened, I'm about 99% sure format-on-save is on

Reply:
whatever it is, the latest of qtcreator borked the formatting, it either doesn't format on save, or it completely messes up the file.
turned that off again, I guess that's a no-go